From c989ce0395e8aea27a440b758c15cf37dc4d2813 Mon Sep 17 00:00:00 2001
From: hyphop <art@khadas.com>
Date: Sat, 15 Feb 2020 15:04:58 +0900
Subject: [PATCH 11/15] change default config for VIMx

activate uboot env from 1st fat partition, we can use saveenv again
---
 configs/khadas-vim2_defconfig  | 5 +++++
 configs/khadas-vim3_defconfig  | 5 +++++
 configs/khadas-vim3l_defconfig | 5 +++++
 configs/khadas-vim_defconfig   | 5 +++++
 4 files changed, 20 insertions(+)

diff --git a/configs/khadas-vim2_defconfig b/configs/khadas-vim2_defconfig
index aa138c892f..ed62da6f89 100644
--- a/configs/khadas-vim2_defconfig
+++ b/configs/khadas-vim2_defconfig
@@ -61,3 +61,8 @@ CONFIG_DM_VIDEO=y
 CONFIG_SYS_WHITE_ON_BLACK=y
 CONFIG_VIDEO_MESON=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_ENV_IS_IN_FAT=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
+CONFIG_ENV_FAT_DEVICE_AND_PART="2"
+CONFIG_ENV_FAT_FILE="uboot.env"
+CONFIG_ENV_SIZE=0x8000
diff --git a/configs/khadas-vim3_defconfig b/configs/khadas-vim3_defconfig
index f73dafae0b..79f5d4a866 100644
--- a/configs/khadas-vim3_defconfig
+++ b/configs/khadas-vim3_defconfig
@@ -59,3 +59,8 @@ CONFIG_USB_GADGET_DWC2_OTG=y
 CONFIG_USB_GADGET_DWC2_OTG_PHY_BUS_WIDTH_8=y
 CONFIG_USB_GADGET_DOWNLOAD=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_ENV_IS_IN_FAT=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
+CONFIG_ENV_FAT_DEVICE_AND_PART="2"
+CONFIG_ENV_FAT_FILE="uboot.env"
+CONFIG_ENV_SIZE=0x8000
diff --git a/configs/khadas-vim3l_defconfig b/configs/khadas-vim3l_defconfig
index dc35e9f148..b78f2dda3b 100644
--- a/configs/khadas-vim3l_defconfig
+++ b/configs/khadas-vim3l_defconfig
@@ -64,3 +64,8 @@ CONFIG_DM_VIDEO=y
 CONFIG_VIDEO_MESON=y
 CONFIG_VIDEO_DT_SIMPLEFB=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_ENV_IS_IN_FAT=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
+CONFIG_ENV_FAT_DEVICE_AND_PART="2"
+CONFIG_ENV_FAT_FILE="uboot.env"
+CONFIG_ENV_SIZE=0x8000
diff --git a/configs/khadas-vim_defconfig b/configs/khadas-vim_defconfig
index 7471085ea4..180ff73e6d 100644
--- a/configs/khadas-vim_defconfig
+++ b/configs/khadas-vim_defconfig
@@ -62,3 +62,8 @@ CONFIG_DM_VIDEO=y
 CONFIG_SYS_WHITE_ON_BLACK=y
 CONFIG_VIDEO_MESON=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_ENV_IS_IN_FAT=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
+CONFIG_ENV_FAT_DEVICE_AND_PART="2"
+CONFIG_ENV_FAT_FILE="uboot.env"
+CONFIG_ENV_SIZE=0x8000
-- 
2.17.1

